packages/cli/generators/import-model/index.js

```diff
@@ -0,0 +1,175 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const chalk = require('chalk');
+const path = require('path');
+const BaseGenerator = require('../../lib/base-generator');
+const modelUtils = require('../../lib/model-discoverer');
+const debug = require('../../lib/debug')('import-model-generator');
+const utils = require('../../lib/utils');
+
+module.exports = class ModelImporter extends BaseGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.argument('modelFile', {
+      type: String,
+      required: true,
+      description:
+        'Path to the model JSON file to import, ' +
+        'e.g. "lb3app/common/models/{model-name}.json"',
```

Comment:
Some metadata are in `model-config.json`, we should read them too.

Reply:
Let's leave that out of the initial implementation, I'd like to avoid scope creep. See the acceptance criteria https://github.com/strongloop/loopback-next/issues/2480.

Reply:
Reworked the code to load the entire app, see https://github.com/strongloop/loopback-next/pull/3688#issuecomment-533070495

This way we take into account any extra configuration provided in `model-config.json`.